### ISSUE TITLE
test_remote_script_module is flaky

### ISSUE THREAD

**Issue by mrshenli** [pytorch/pytorch#33948]
Failed run:
https://app.circleci.com/jobs/github/pytorch/pytorch/4605753

Passed run:
https://app.circleci.com/jobs/github/pytorch/pytorch/4605780

cc @ezyang @gchanan @zou3519 @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @rohan-varma @xush6528 @jjlilley @osalpekar

**Comment by zhaojuanmao** [comment]
the error is "Received error while processing request type 4: vector::_M_range_check: __n (which is 0) >= this->size() (which is 0)", I checked this test, did not find suspicious causes. Also can not reproduce locally in stress runs, I checked master runs as well, it only happened once. I suspect it is due to some transient failure that is not related to our code bases. 

I will keep monitoring this error.
